lib/src/layer.dart

```diff
@@ -26,6 +26,12 @@ class Layer {
 
   Layer(this.name, this.width, this.height);
 
+
+  @override
+  String toString() {
```

Comment:
I am personally not a fan of using `toString` methods for debugging reasons. If I want to know whats inside an instance I'll just add breakpoint in my debugger.

And now we also have to keep this uptodate. So my personal preference would be to remove them.

What do you think @luanpotter ?

Reply:
I use this toString as an easy method to check the results in parser_compare_test.dart
But maybe it would be better to create equals methods... but they had to be changed also, every time the class is changed. What do you think is better?
Or should i make a "big test" and compare every attribute?

Reply:
If the default equal methods are not sufficient for comparisons I would recommend overwriting the equals methods.

Reply:
cant you just assert the properties?
```
   assertEquals(layer.width, 100)
```
etc

Reply:
this is an old version and the toString is deleted